packages/veritone-react-common/src/components/GeoPicker/index.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import GeoPicker from './';
+
+describe('GeoPicker', () => {
+
+  const gmapsAPIKey = 'AIzaSyDQyglufOtb_uzebFNkVWcB1D8-tBKZarQ';
```

Comment:
secret? should probably put this in the storybook config

Reply:
Not a secret, it's whitelisted by origin. Do you still want to add it to storybook config?